android/java/org/chromium/chrome/browser/local_database/DatabaseHelper.java

```diff
@@ -53,6 +56,7 @@ public void onCreate(SQLiteDatabase db) {
         db.execSQL(TopSiteTable.CREATE_TABLE);
         db.execSQL(BraveStatsTable.CREATE_TABLE);
         db.execSQL(SavedBandwidthTable.CREATE_TABLE);
+        db.execSQL(DisplayAdsTable.CREATE_TABLE);
```

Comment:
I was going to ask, what will happenif the table already exists, but then I found it is handled in SQL, `CREATE TABLE IF NOT EXISTS ....`  :+1: